.github/workflows/nodejs-ci.yml

```diff
@@ -248,6 +244,9 @@ jobs:
         id: dependency_scan
         if: ${{ always() && !contains( inputs.excluded_jobs, 'dependency_scan' ) }}
         run: |
+          cat >> /root/configs/osv/config.toml <<EOF
+              ${{ vars.OSV_SUPPRESSION_ID }}
+          EOF
```

Comment:
Does this work if you give list of IDs in variable?

Reply:
Variable must be added in readme in a separate section `Action variables`. Ref to README-checkov-terraform-iac-scan.md